README.md

```diff
@@ -545,6 +579,32 @@ types.py:5: error: Argument 1 to "foo" has incompatible type "int"; expected "st
 Found 2 errors in 1 file (checked 1 source file)
 ```
 
+### TypedDict
+
+As of 3.8, python's `typing` module provides the `TypedDict` class, allowing
+you to specify the member types and key names of a `dict. This is useful for
+declaring data classes as a distinct type, while still affording the ease of
+use of a conventional dict. From the docs:
+
+> TypedDict declares a dictionary type that expects all of its instances to
+> have a certain set of keys, where each key is associated with a value of a
+> consistent type. This expectation is not checked at runtime but is only
+> enforced by type checkers.
+
+Here's how to use it:
+
+```python3
+>>> import typing
+>>> class MyData(typing.TypedDict):
+...     a: int
+...     b: int
+...
+>>> MyData(a=1, b=2)
+{'a': 1, 'b': 2}
```

Comment:
usually you actually dont use it directly (but you could). you simply declare it in type annotation and then us regular dict:

```suggestion
>>> def foo(data: MyData) -> MyData:
...     return data
>>> foo({"a": 0, "b": 0}) # passes type checks
>>> foo({"foo": 0})       # fails type checks
```

feel free to add this as example to `types.py` which is showcased how to use a little above. youll need to update the bash output though

Reply:
i see, that makes sense. will update. thanks!